tests/test_debugging.py

```diff
@@ -591,7 +593,11 @@ def circuit():
         _compare_numpy_dicts(result, expected)
 
         # Make sure shots are overridden correctly
-        result = qml.snapshots(circuit)(shots=200)
+        with pytest.warns(
+            PennyLaneDeprecationWarning,
+            match="'shots' specified on call to a QNode is deprecated",
+        ):
+            result = qml.snapshots(circuit)(shots=200)
         assert result[3] == {"0": 98, "1": 102}
         assert np.allclose(result[5], expected[5])
```

Comment:
Since `snapshots` has nothing to do with setting shots, can't we remove this part completely? Why are we testing shot setting here?